### PR TITLE
feat: add lightning address to homescreen

### PR DESCRIPTION
**Pull request by reneaaron**
### Describe the changes you have made in this PR

image

### CODE REVIEW

**Review by rolznz — APPROVED** [review]
utACK

File: src/app/screens/Home/DefaultView/index.tsx [hunk 1]
```diff
@@ -53,6 +57,18 @@ const DefaultView: FC<Props> = (props) => {
...
+  async function getLightningAddress() {
+    setLoadingLightningAddress(true);
+    const response = await api.getAccountInfo();
+    const lightningAddress = response.info.lightning_address;
+    if (lightningAddress) setLightningAddress(lightningAddress);
+    setLoadingLightningAddress(false);
+  }
+
+  useEffect(() => {
+    getLightningAddress();
```
Comment (rolznz) [inline src/app/screens/Home/DefaultView/index.tsx:69]: should this be a hook? `useLightningAddress()`?

Comment (reneaaron) [reply src/app/screens/Home/DefaultView/index.tsx:69]: Yes, good idea! :+1:

**Comment by bumi** [comment]
@reneaaron does this contain some unit test issues?

**Pull request merged**